docs/web/docs/1-getting_started/1-getting_started.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Install and Configure
+title: Getting started
```

Comment:
Maybe: Getting Started ?

Reply:
Thanks @castrapel , now that I think about it, what about we make this "Configuring IAMbic", or "Setting Things Up", since the upper section is already called "Getting Started". 
Then the rest of the inner sections could be:
- Configuring AWS IAM
- Configuring Azue AD IAM
- etc ...

What do you think?